easybuild/easyconfigs/r/R/R-4.3.2-gfbf-2023b.eb

```diff
@@ -0,0 +1,375 @@
+name = 'R'
+version = '4.3.2'
```

Comment:
Final release of R v4.3.3 is scheduled for 29 Feb'24, see news section in https://www.r-project.org .

Can we wait until it's out, and use that (also to avoid having the same version as in the `2023a` generation of easyconfigs)?